src/truncated/loguniform.jl

```diff
@@ -0,0 +1,9 @@
+truncated(d::LogUniform, lo::T, hi::T) where {T<:Real} = truncated_LogUniform(d,lo,hi)
+
+function truncated_LogUniform(d::LogUniform, lo,hi)
+    a, b = params(d)
+    return LogUniform(max(a, lo), min(b, hi))
+end
+
+# fix method ambiguity
+truncated(d::LogUniform, lo::Integer, hi::Integer) = truncated_LogUniform(d,lo,hi)
```

Comment:
If you merge the latest changes from the master branch, you should be able to change this to:
```suggestion
truncated(d::LogUniform, lo::T, hi::T) where {T<:Real} = LogUniform(max(d.a, lo), min(d.b, hi))
```